### ISSUE TITLE
Added sun, moon, and local time

### ISSUE THREAD

**Issue by bHogan2** [ThingPulse/esp8266-weather-station#32]
Just an FYI / suggestion.

I added "Astronomy" to updateData and made frames for sunrise / sunset times and phase of the moon.

The bonus is that the astronomy record also provides the local time - adjusted to DST and TZ. This eliminates the need to use TimeClient and do the conversions.

I'd be happy to share the changes to the current version of WundergroundClient.cpp and .h if you are interested.

**Comment by squix78** [comment]
That would be great! Easiest for me would be a pull request: just fork this repository, apply your changes and create a pull request with the differences between your fork and the master repository. Or if you prefer, attach your files  to this issue...

**Comment by bHogan2** [comment]
Sorry, I'd like to do it the easiest way for you but I'm afraid I'm GitHub impaired :)
I've attached the files. The changes are all commented, just search for "JJG".

I apologize in advance for the hack job I did with returning AM & PM. I was too excited and scared to do it right!

I also added screenshots of the sun and moon frames just to give you the idea. The Meteocons I used were "1" and "J".

Thanks again for a very well put together project. I had a nice time with it.
(I can be reached at the contact info on my blog - http://brohogan.blogspot.com/)
[sun+moon.zip](https://github.com/squix78/esp8266-weather-station/files/299436/sun.moon.zip)
